app/components/Chatbot/chat_entry.css

```diff
@@ -122,10 +122,11 @@ article.stampy {
   word-wrap: break-word;
   background-color: white;
   border: 1px solid var(--colors-cool-grey-200, #dfe3e9);
-  padding: var(--spacing-24) var(--spacing-32);
+  padding: var(--spacing-40);
   position: absolute;
   transform: translateX(50%);
   text-decoration: unset;
+  z-index: 3;
```

Comment:
shouldn't the element just have the `z-index-3` class?

Reply:
Yeah probably